Externals.cfg

```diff
@@ -25,10 +25,10 @@ required = True
 
 [ufs_weather_model]
 protocol = git
-repo_url = https://github.com/ufs-community/ufs-weather-model
+repo_url = https://github.com/NOAA-GSD/ufs-weather-model
 # Specify either a branch name or a hash but not both.
 #branch = dtc/develop
```

Comment:
I know that this is a comment, but it is no longer consistent with the url/hash. Do you mind updating it to say "gsd/develop"?